Neos.CliSetup/Classes/Command/WelcomeCommandController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\CliSetup\Command;
+
+/*
+ * This file is part of the Neos.CliSetup package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Cli\CommandController;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class WelcomeCommandController extends CommandController
+{
+
+    public function indexCommand(): void
+    {
+        $this->output(
+            <<<EOT
+            <info>
+                ....######          .######
+                .....#######      ...######
+                .......#######   ....######
+                .........####### ....######
+                ....#......#######...######
+                ....##.......#######.######
+                ....#####......############
+                ....#####  ......##########
+                ....#####    ......########
+                ....#####      ......######
+                .#######         ........
+
+            Welcome to Neos.
+            </info>
+
+            The following steps will help you to configure your Neos:
```

Comment:
Suggestion Remove "Your" so the sentence goes

`The following steps will help you to configure Neos`

Reply:
Applied as suggested thanks